api/managementpb/checks.proto

```diff
@@ -28,6 +29,38 @@ message SecurityCheckResult {
   string service_name = 6;
 }
 
+// CheckResultSummary is a summary of check results.
+message CheckResultSummary {
+  string service_name = 1;
+  string service_id = 2;
+  // Number of failed checks for this service that are critical
+  uint32 critical_count = 3;
+  // Number of failed checks for this service that are major
+  uint32 major_count = 4;
+  // Number of failed checks for this service that are trivial
+  uint32 trivial_count = 5;
+}
+
+// CheckResult represents the check results for a given service.
+message CheckResult {
+  string summary = 1;
+  string description = 2;
+  Severity severity = 3;
+  map<string, string> labels = 4;
```

Comment:
And where is annotations?

Reply:
I think all the fields in the annotations are already in the message directly (summary, name and description). With labels, it helps to send all of it back so that they can be used for filtering, etc